src/main/java/com/spartronics4915/frc2020/Constants.java

```diff
@@ -55,7 +58,8 @@
         public static final int kOpticalFlagId = 2; // A2 // Optical Flag for Zeroing
         public static final boolean kOpticalFlagReversed = false; // Whether or not the optical flag
                                                                   // is inverted
-        public static final int kIntakeSensorId = -1; // 
+        //TODO: sounds misplaced…needs a better name?
+        public static final int kIntakeSensorId = -1;
```

Comment:
This sensor has a port assigned to it - check the [2020 Control Map spreadsheet.](https://docs.google.com/spreadsheets/d/14O3Dj1Bl6tqiHFrCJ7wyVRDOqS5EZ11p4q9j2rYS-3Q/edit)

If you want a different name, I'd suggest `kProximitySensorId`.